test/utils/snap.go

```diff
@@ -169,6 +169,22 @@ func SnapRefresh(t *testing.T, name, channel string) {
 	), true)
 }
 
+func SnapServicesStartup(t *testing.T, name string) string {
```

Comment:
This function can do more internally and only return the boolean, instead of letting the caller do string comparison:
```suggestion
func SnapServicesEnabled(t *testing.T, name string) bool {
```